### ISSUE TITLE
Can't access database 

### ISSUE THREAD

**Issue by Helen300** [nickfox/GpsTracker#104]
Hello, I'm a beginner programmer trying to set up the MySQL database with the php server, but I'm having trouble accessing the database. Whenever I go to http://localhost:8080/gpstracker/php/updatelocation.php

I get this error and I'm not sure how to debug this. 

Fatal error: Uncaught PDOException: SQLSTATE[HY000] [1045] Access denied for user 'gpstracker_user'@'localhost' (using password: YES) in /opt/lampp/htdocs/gpstracker/php/dbconnect.php:24 Stack trace: #0 /opt/lampp/htdocs/gpstracker/php/dbconnect.php(24): PDO->__construct('mysql:host=loca...', 'gpstracker_user', 'gpstracker') #1 /opt/lampp/htdocs/gpstracker/php/updatelocation.php(2): include('/opt/lampp/htdo...') #2 {main} thrown in /opt/lampp/htdocs/gpstracker/php/dbconnect.php on line 24

**Comment by coose2018** [comment]
I'm curious does it function yet?

**Comment by brentfraser** [comment]
I think Nick considers this project to be archived and he may not be monitoring it anymore, but I'll try to help.

The obvious question is can you access your database using the user name and password stored in dbconnect.php?

**Comment by coose2018** [comment]
Sure that works to. But if its shelved it doesn't work it appears not a lot
has been done either in a good bit

**Comment by coose2018** [comment]
Well Brent hes onto a great idea that's an actual need with the new phones
impossible to get into now

**Comment by coose2018** [comment]
I didnt know I had a password anymore I haven't been there in so long and
came across that project. I had to see if it was complete

**Comment by brentfraser** [comment]
I'm confused.  Does the "gpstracker_user" in your MySQL database have a password?  Did you follow the instructions on https://www.websmithing.com/2014/01/20/quick-start-guide-for-gpstracker-3/ ?

**Comment by coose2018** [comment]
Not that I'm aware of u mean the zip I grabbed I thought it was either
finished or close to it. I think maybe i should bring it back i wasnt aware
of a password that's not cool i can i opened it and well whatever it said
was a good thing but it's one part to the bigger picture. I'm sorry dude

**Comment by coose2018** [comment]
Yes I followed the instructions but dont recall a password am I getting
someone in trouble here I hope not I just got a zip file that I figured
wasnt complete

**Comment by brentfraser** [comment]
The zip file is fine.  There are two parts to GPSTracker: one is the mobile phone app (you can install it from the Play Store).  The other is the server code.  You install it on your server by unzipping the zip file and configuring it by modifying the dbconnect.php file to point to your server.

The Quick Start Guide says you must create a username and password for your MySQL database.  Did you do that?

**Comment by coose2018** [comment]
No I didnt get the second part we so it does work why are u not doing this
or do u have top secret going on lol

**Comment by coose2018** [comment]
So does it work or is that up to me to figure out the instructions are
there all of it

**Comment by brentfraser** [comment]
It works fine if you follow the instructions to set up your server.

**Comment by coose2018** [comment]
Ok thank u very much that's to cool I appreciate all your help
